client/deis.py

```diff
@@ -1302,6 +1302,19 @@ def config_unset(self, args):
         else:
             raise ResponseError(response)
 
+    def _read_config_from_path(self, path):
+        env_dict = {}
+        with open(path, 'r') as f:
+            for line in f:
+                line = line.strip()
+                if re.match(r'.+=.+', line) is None:
+                    self._logger.warning('could not parse config line: "%s"', line)
```

Comment:
If we have an empty line as explained in #3658, we should skip this warning. For example:

```
$ cat .env
FOO=bar

CAR=star
```

`could not parse config line: ""`